examples/platform/linux/AppMain.cpp

```diff
@@ -1017,6 +1017,9 @@ void ChipLinuxAppMainLoop(AppMainLoopImplementation * impl)
     sigaction(SIGTERM, &sa, nullptr);
 #endif
 
+    // This message is used as a marker for when the application process has started.
+    // See: scripts/tests/chiptest/test_definition.py
+    ChipLogProgress(DeviceLayer, "Starting event loop");
```

Comment:
nit: if we do this maybe we should make it a bit more magical, like `===== APP STATUS: Starting event loop =====` or similar, to avoid the chance of others writing similar messages.

Beyond that, should we add some TODO that for linux we should use some other messaging method? We are looking at doing this for OTA and stop log scraping, yet here we are again log scraping.

I am find as a followup, however in practice I believe we should have something more stable, since we are on linux for example we could use named pipes (and then tests can wait for named pipe appearance and then use that) or lock files that we give through command line or any other real IPC mechanism.